Content.Shared/GameObjects/Components/Body/SharedBodyComponent.cs

```diff
@@ -697,6 +702,19 @@ public override void HandleComponentState(ComponentState? curState, ComponentSta
                 }
             }
         }
+
+        public virtual void Gib(bool recursive = false)
```

Comment:
`bool gibParts` would be more descriptive here, and would also allow for `gibMechanisms` at a future date.

Reply:
Done.